src/cc/usdt.h

```diff
@@ -159,6 +164,8 @@ class Probe {
 
 class Context {
   std::vector<Probe *> probes_;
+  std::vector<std::pair<Probe *, std::string>> uprobes_;
```

Comment:
Can the `probes_` and `uprobes_` vectors contain `std::unique_ptr<Probe>` instead of raw pointers?